### DISCUSSION TITLE
Why I Can’t Find Github App on Play Store?

### DISCUSSION THREAD

**Discussion by coderspoty** [community/community#81428] [Mobile] [answered]
### Select Topic Area

Question

### Body

Github has a Github App, I heard from my friend. However, when I write to the Google Play to download it to my phone, the app doesn’t appear. Btw, my friend downloaded the app from the App Store. Why can't I see Github app on Google Play?

**Comment by hopesf** [comment, marked as answer]
The reason why you can't find the Github app from Google Play is most likely because the app's creators have designated your device as "incompatible" with it. Do you utilize an outdated Android version? Is your equipment outdated? These can be the causes of your inability to locate the app. You must fulfill some prerequisites in order to find an app, such as having a device with sufficient processing capability to execute the app. I advise you to get in touch with Github to see if they can update the app to work with your particular device.
